packages/app/src/components/EventList.js

```diff
@@ -1,12 +1,14 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 import {ScrollView, Text} from 'react-native';
 import {graphql} from 'babel-plugin-relay/macro';
 import {QueryRenderer} from 'react-relay';
 import Environment from '../relay/Environment';
 import EventCard from './EventCard';
+import EventCreateSubscription from './EventCreateSubscription';
 
 const EventList = ({query}) => {
   const {events} = query;
+  useEffect(() => EventCreateSubscription(), []);
```

Comment:
is this necessary?

Reply:
That's how it's made in this [tutorial](https://www.howtographql.com/react-relay/7-subscriptions/), but with `componentDidMount` instead.